tests/test_mha.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+import torch
+
+from i6_models.parts.mha import MultiheadAttentionV1, MultiheadAttentionV1Config
+
+
+def test_MultiheadAttentionV1():
```

Comment:
I know this was already mentioned offline, but just for reference, we also need a test to compare against the PyTorch MultiheadAttention

Reply:
Added.